packages/libs/tsconfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "extends": "@tsconfig/strictest/tsconfig.json",
   "compilerOptions": {
-    "lib": ["ES2020", "dom"],
-    "target": "es5",
+    "module": "ES2022",
```

Comment:
for verify to pass, i had to do this. Not entirely certain of the implications... cc: @sliptype / @dylanjeffers fyi

Reply:
I think this would mainly affect the vanilla js bundle, essentially anyone that isn't using a bundler could no longer use sdk in older browers. But anyone that is bundling can still transpile down and polyfill things for older browsers. So we should be good

Also es5 is just a relic of the initial version of libs, so this upgrade needed to happen at some point anyway